docs/user/02-operation-guides/operations/02-20-enable-sidecar-injection.md

```diff
@@ -2,46 +2,51 @@
-
-* To enable sidecar proxy injection for a Namespace, set the **istio-injection** label value to `enabled` for the Namespace in which you want to enable the sidecar proxy injection. Use the following command:
-
-   ```bash
-   kubectl label namespace {YOUR_NAMESPACE} istio-injection=enabled
-   ```
-
-* To enable sidecar proxy injection for a Deployment, add this label to the Deployment configuration file: `sidecar.istio.io/inject: "true"`.
-
-Note that if the sidecar proxy injection is disabled at the Namespace level or the `sidecar.istio.io/inject` label on a Pod is set to `false`, the sidecar proxy is not injected.
-
-## Check whether your workloads have automatic Istio sidecar injection enabled
-
-Check whether your workloads have automatic Istio sidecar injection enabled by running [this script](../../../assets/sidecar-analysis.sh). You can either pass the **namespace** parameter to the script or run it with no parameter.
-
-If you don't provide any parameter, the execution output contains Pods from all Namespaces that don't have automatic Istio sidecar injection enabled. If you pass a parameter, only the Pods from the specified Namespace are analyzed.
-
-The script outputs the information in the format of `{namespace}/{pod}` if run for all Namespaces and in the format of `{pod}` if run for a specific Namespace. Run the script:
-
-```bash
-./sidecar-analysis.sh {namespace}
-```
-
-
-* If you run `./sidecar-analysis.sh`, you get an output similar to this one:
-
-```
-Pods out of istio mesh:
-  In namespace labeled with "istio-injection=disabled":
-    - sidecar-disabled/some-pod
-  In namespace labeled with "istio-injection=enabled" with pod labeled with "sidecar.istio.io/inject=false":
-    - sidecar-enabled/some-pod
-  In not labeled ns with pod not labeled with "sidecar.istio.io inject=true":
-    - no-label/some-pod
-```
-
-* If you run `./sidecar-analysis.sh some-namespace`, you get an output similar to this one:
-
-```
-Pods out of istio mesh in namespace some-namespace:
-  - some-pod
-```
+You can enable sidecar proxy injection for either an entire Namespace or a single Deployment. If the sidecar proxy injection is disabled at the Namespace level or the `sidecar.istio.io/inject` label on a Pod is set to `false`, the sidecar proxy is not injected.
+
+>**WARNING:** Adding the `istio-injection=enabled` label on the Namespace level results in injecting sidecars to all Pods inside of the Namespace. 
+
+* Follow the steps to enable sidecar proxy injection for a Namespace:
+  
+  <!-- tabs:start -->
+
+  #### **kubectl**
+  
+  Use the following command:
+  
+  ```bash
+  kubectl label namespace {YOUR_NAMESPACE} istio-injection=enabled
+  ```
+
+  #### **Kyma Dashboard**
+  
+  1. Select the Namespace where you want to enable sidecar proxy injection.
+  2. Click the **Edit** button.
+  3. In the `UI Form` section, toggle the switch to set the **istio-injection** label value to `enabled` for the Namespace.
```

Comment:
```suggestion
  3. In the `UI Form` section, set the **istio-injection** label value to `enabled` for the Namespace.
```
Again, I'd rather be slightly more abstract than describing the UI details; because they might change